content/sensu-go/5.16/guides/using-check-aggregates.md

```diff
@@ -0,0 +1,285 @@
+  subscriptions:
+  - linux
+```
+
+```yaml
+---
+type: CheckConfig
+metadata:
+  name: check-webapp-8080
+  namespace: default
+spec:
+  command: "check-http.rb -u --port 8080 http://webserver01.example.com"
+  handlers: 
+  - slack
+  high_flap_threshold: 0
+  interval: 10
+  low_flap_threshold: 0
+  publish: true
+  runtime_assets:
+  - sensu-plugins/sensu-plugins-http
+  - sensu/sensu-ruby-runtime
+  subscriptions:
+  - linux
+```
+
+```yaml
+---
+type: CheckConfig
+metadata:
+  name: check-webapp-9000
+  namespace: default
+spec:
+  command: "check-http.rb -u --port 9000 http://webserver01.example.com"
+  handlers: 
+  - slack
+  high_flap_threshold: 0
+  interval: 10
+  low_flap_threshold: 0
+  publish: true
+  runtime_assets:
+  - sensu-plugins/sensu-plugins-http
+  - sensu/sensu-ruby-runtime
+  subscriptions:
+  - linux
+```
+
+So we have three separate checks that are monitoring our web application. However, if we wanted to view the webapp's health these three checks don't do the best job providing that insight. The checks are isolated from each other and each check alerts individually. 
+
+Instead, it makes more sense to have this group of checks be part of an aggregate--we might not care if a check on an individual host fails, but we certainly care if a large percentage of the checks are in a warning or critical state across a number of hosts.We can do this by adding a label to the checks:
+
+```yaml
+---
+type: CheckConfig
+metadata:
+  name: check-webapp-80
+  namespace: default
+  labels:
+    service_type: webapp
+spec:
+  command: "check-http.rb -u http://webserver01.example.com"
+  high_flap_threshold: 0
+  interval: 10
+  low_flap_threshold: 0
+  publish: true
+  runtime_assets:
+  - sensu-plugins/sensu-plugins-http
+  - sensu/sensu-ruby-runtime
+  subscriptions:
+  - linux
+```
+
+```yaml
+---
+type: CheckConfig
+metadata:
+  name: check-webapp-8080
+  namespace: default
+  labels:
+    service_type: webapp
+spec:
+  command: "check-http.rb -u --port 8080 http://webserver01.example.com"
+  high_flap_threshold: 0
+  interval: 10
+  low_flap_threshold: 0
+  publish: true
+  runtime_assets:
+  - sensu-plugins/sensu-plugins-http
+  - sensu/sensu-ruby-runtime
+  subscriptions:
+  - linux
+```
+
+```yaml
+---
+type: CheckConfig
+metadata:
+  name: check-webapp-9000
+  namespace: default
+  labels:
+    service_type: webapp
+spec:
+  command: "check-http.rb -u --port 9000 http://webserver01.example.com"
+  high_flap_threshold: 0
+  interval: 10
+  low_flap_threshold: 0
+  publish: true
+  runtime_assets:
+  - sensu-plugins/sensu-plugins-http
+  - sensu/sensu-ruby-runtime
+  subscriptions:
+  - linux
+```
+
+Each check now has a label that we can use as part of an aggregate that gives us more visibility into the health of our webapp. You'll also notice that we removed handlers from the check--if we want to alert on an aggregate, we're better served having the aggregate handled versus having each individual check handled.
```

Comment:
```suggestion
You can use the labels as part of an aggregate that gives you more visibility into the health of our webapp. You also removed handlers from the check. If you want to alert on an aggregate, it's better to handle the aggregate instead of handling each individual check.
```